build-tokens.js

```diff
@@ -0,0 +1,134 @@
+const _ = require("lodash")
```

Comment:
we should delete this and all other files from inside the app folder too. Better yet would be to move them in this position